### PR TITLE
[JENKINS 63160] Fix .pane-header text and background colors

### PR DESCRIPTION
**Pull request by fqueiruga**
See [JENKINS-63160](https://issues.jenkins-ci.org/browse/JENKINS-63160).

This PR fixes an issue introduced in https://github.com/jenkinsci/jenkins/pull/4835 where the `.pane-header` text and background colors would result in an impossible to read element.

The solution is to remove the dependency of the bigtable and tab header colors from the `.pane-header`, and then fix the missing variables.

The issue was discovered on the Matrix Auth plugin.

<details>
<summary>Before</summary>

Screenshot 2020-07-22 at 11 45 15
</details>

<details>
<summary>After</summary>

Captura de pantalla 2020-07-22 a las 11 52 50
</details>

### Proposed changelog entries

* Fix an issue where the header of certain elements such as the authorization matrix would have wrong styles.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@daniel-beck 
@timja 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Looks right :+1:

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by fqueiruga** [comment]
Please do not merge until I fix something. This fix messed up the test results table:

Captura de pantalla 2020-07-23 a las 15 46 58

Gotta love how no table is done the same way :S

**Comment by fqueiruga** [comment]
Fixed! It should be good to go.

Captura de pantalla 2020-07-23 a las 15 52 28

**Review by oleg-nenashev — APPROVED** [review]
Ship it then?

**Pull request merged**
